tools/migrate/migrate_subscription.ts

```diff
@@ -14,7 +14,7 @@ async function main() {
   await conn.ping();
   const db = mysqlDrizzle(conn, { schema, mode: "default" });
 
-  const workspaceId = "ws_39g5eLLQTX8bVdbsGK9Dke";
+  const workspaceId = "ws_wB4SmWrYkhSbWE2rH61S6gMseWw";
```

Comment:
Do you want to set this to WORKSPACE_ID versus an actual workspace?

Reply:
ah this one is unrelated, I just recently migrated this user, ignore pls